index.js

```diff
@@ -27,20 +43,19 @@ const context = {
 };
```

Comment:
on line 41, https://github.com/launchdarkly/sdk-specs/tree/main/specs/EXAM-SDK-example/HELLO-hello-apps#requirement-121 says this should be:
```
key: "example-user-key",
```